README.md

```diff
@@ -62,7 +66,9 @@ Owing to this dependency on Yarn Workspaces, Create Eth App can't be used with n
 
 In the project directory, you can run:
 
```

Comment:
We should remove the subgraph documentation in this README, since there is no subgraph in this repository.

Also, we don't need to document `react-app:eject`, it's fairly common for `create-react-app` (and therefore `create-eth-app`) and we probably won't ever use it.